terraform/performance/main.tf

```diff
@@ -55,16 +61,18 @@ resource "aws_instance" "cwagent" {
   ami                         = data.aws_ami.latest.id
   instance_type               = var.ec2_instance_type
   key_name                    = local.ssh_key_name
-  iam_instance_profile        = data.aws_iam_instance_profile.cwagent_instance_profile.name
-  vpc_security_group_ids      = [data.aws_security_group.ec2_security_group.id]
+  iam_instance_profile        = module.basic_components.instance_profile
+  subnet_id                   = module.basic_components.random_subnet_instance_id
+  vpc_security_group_ids      = [module.basic_components.security_group]
   associate_public_ip_address = true
+
   metadata_options {
     http_endpoint = "enabled"
     http_tokens   = "required"
   }
 
   tags = {
-    Name = "cwagent-performance-${var.test_name}-${module.common.testing_id}"
```

Comment:
Why do we not want the test name?

Reply:
Same comment as previous comment a little bit https://github.com/aws/amazon-cloudwatch-agent-test/pull/166#discussion_r1135635656